quad/src/constants.rs

```diff
@@ -32,6 +33,26 @@ pub fn add_res(v: &mut [f64], w: &[f64]) {
     }
 }
 
+pub fn points_transformed(mut points: Vec<f64>, a: f64, b: f64) -> Vec<f64> {
+    points.sort_by(|a, b| a.partial_cmp(b).unwrap());
+    let mut points_transformed = vec![0.0; 0];
+    if b == f64::INFINITY && a.is_finite() {
+        for point in &points {
```

Comment:
You can swap the `for` loop and the `if`, since the loop is the exact same for all of them.